src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx

```diff
@@ -220,40 +216,22 @@ const Room: React.FC<RoomProps> = ({
     ? worldUsersById[room.localParticipant.identity]
     : undefined;
 
-  const participantContainerClassName = useMemo(() => {
-    const attendeeCount = (participants.length ?? 0) + 1; // Include yourself
-    if (attendeeCount <= 4) {
-      return "two-across";
-    }
-    return "three-across";
-  }, [participants.length]);
+  const [sidedVideoParticipants, otherVideoParticipants] = useMemo(() => {
+    const sidedVideoParticipants = participants.slice(
+      0,
+      NUM_OF_SIDED_USERS_MINUS_ONE
+    );
 
-  const meComponent = useMemo(() => {
-    return room && profileData ? (
-      <div
-        className={`jazzbar-participant-container ${participantContainerClassName}`}
-      >
-        <LocalParticipant
-          key={room.localParticipant.sid}
-          participant={room.localParticipant}
-          profileData={profileData}
-          profileDataId={room.localParticipant.identity}
-          bartender={meIsBartender}
-          defaultMute={defaultMute}
-        />
-      </div>
-    ) : null;
-  }, [
-    meIsBartender,
-    room,
-    profileData,
-    defaultMute,
-    participantContainerClassName,
-  ]);
-
-  const othersComponents = useMemo(
+    const otherVideoParticipants = participants.slice(
+      NUM_OF_SIDED_USERS_MINUS_ONE
+    );
+
+    return [sidedVideoParticipants, otherVideoParticipants];
+  }, [participants]);
```

Comment:
The logic is purely for having nice alignment for the bottom row of videos. Grid doesn't allow that, so we have grid for sided videos and flex for otherVideos

Reply:
While I haven't checked this specifically, this issue I stumbled across suggests that it could be possible without this:

- https://stackoverflow.com/questions/43129360/css-grid-wrapping/43129507#43129507

![image](https://user-images.githubusercontent.com/753891/109809319-45934380-7c7c-11eb-808c-af7fc5f96efe.png)

(not a blocker for this PR, but if it works, is probably a cleaner pattern for future/similar usecases)

Reply:
I don't think it is that easy. I guess this was my first approach. It seems like grid doesn't offer an easy solution for this. Mentioned this dicussion in https://github.com/sparkletown/internal-sparkle-issues/issues/501 though